src/stream.ts

```diff
@@ -131,6 +102,30 @@ export class WebRTCStream implements Stream {
       let err = (evt as RTCErrorEvent).error;
       this.abort(err);
     };
+
+    const self = this;
+    // reader pipe
+    this.channel.onmessage = async ({data}) => {
+      const res = new Uint8Array(data as ArrayBuffer);
```

Comment:
if the length is `0` here, would that not mean that data is some kind of null value as well? Can't we just check that early and return already then, without first alloc'ing an Uint8Array here first?